.github/workflows/master_rancher_ui_experimental_workflow.yml

```diff
@@ -128,15 +98,14 @@ jobs:
         # EXTRAENV_VALUE: 12345
       options: --add-host ${{ needs.installation.outputs.MY_HOSTNAME }}:${{ needs.installation.outputs.MY_IP }} --ipc=host ${{ inputs.docker_options }}
```

Comment:
`inputs.docker_options` (shouldn't it be `github.event.inputs.docker_options`?) not set but needed for firefox browser (value `--user 1000`). Please expand `workflow_dispatch.inputs` with the following block:

```
        docker_options:
          description: Extra options for docker (set to "--user 1000" for firefox)
          required: false
          type: string
          default: ''
```